modules/cluster-logging-eventrouter-deploy.adoc

```diff
@@ -105,27 +105,28 @@ objects:
               configMap:
                 name: eventrouter
 parameters:
-  - name: IMAGE
+  - name: IMAGE <6>
     displayName: Image
-    value: "registry.redhat.io/openshift-logging/eventrouter-rhel8:latest"
-  - name: CPU  <6>
+    value: "registry.redhat.io/openshift-logging/eventrouter-rhel8:v0.3"
+  - name: CPU  <7>
     displayName: CPU
     value: "100m"
-  - name: MEMORY <7>
+  - name: MEMORY <8>
     displayName: Memory
     value: "128Mi"
   - name: NAMESPACE
     displayName: Namespace
-    value: "openshift-logging" <8>
+    value: "openshift-logging" <9>
 ----
 <1> Creates a Service Account in the `openshift-logging` project for the Event Router.
 <2> Creates a ClusterRole to monitor for events in the cluster.
 <3> Creates a ClusterRoleBinding to bind the ClusterRole to the service account.
 <4> Creates a config map in the `openshift-logging` project to generate the required `config.json` file.
 <5> Creates a deployment in the `openshift-logging` project to generate and configure the Event Router pod.
-<6> Specifies the minimum amount of memory to allocate to the Event Router pod. Defaults to `128Mi`.
-<7> Specifies the minimum amount of CPU to allocate to the Event Router pod. Defaults to `100m`.
-<8> Specifies the `openshift-logging` project to install objects in.
+<6> Specifies the image, identified by the tag (this example uses the v0.3 tag).
```

Comment:
The [ISG](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses) says to avoid using parentheses in general text. What do you think of "Specifies the image, identified by a tag such as `v0.3`."

Reply:
Ah, OK! That is a good way to rephrase, I'll make that change!